packages/mint-components/src/components/sqm-instant-access-registration/sqm-instant-access-registration.feature

```diff
@@ -0,0 +1,118 @@
+@author:truman
+Feature: Instant access referrer registration
+
+    @motivating
+    Scenario: Form has content validation for email input
+        Given a user is trying to register as a referrer
+        When they <error action>
+        And they submit the form
+        Then an error tells the user <message>
+            Examples
+            | error action            | message                    |
+            | do not provide an email | that email cannot be empty |
+            | give an invalid email   | the email is invalid       |
+
+    @motivating
+    Scenario: Name fields are required when included
```

Comment:
This is not the correct behavior, the first name, and last name should not be required. Please change the specs and the component as well.